codalab/apps/web/tests/test_competitions.py

```diff
@@ -155,7 +155,7 @@ def setUp(self):
         self.participant = CompetitionParticipant.objects.create(
             user=self.participant_user,
             competition=self.competition,
-            status=ParticipantStatus.objects.get_or_create(name='approved', codename=ParticipantStatus.PENDING)[0]
+            status=ParticipantStatus.objects.get_or_create(name='approved', codename=ParticipantStatus.APPROVED)[0]
```

Comment:
Yes: for the test `test_can_edit_competition_with_ownership` it need to have a ParticipantStatus.APPROVED created in the DB. What I could do as an alternative is to create another `ParticipantStatus` with a status `APPROVED` so it would be available during the test. But I looked at all the tests and the status PENDING did not seem to be used so I just changed it.
If I missed something let me know

Reply:
This makes sense, I was more of wondering if it was passing before without the status set to approved and if that had changed.